recommend/policy.go

```diff
@@ -89,7 +90,15 @@ func (img *ImageInfo) createPolicy(ms MatchSpec) (types.KubeArmorPolicy, error)
 
 	// add container selector
 	repotag := strings.Split(img.RepoTags[0], ":")
-	policy.Spec.Selector.MatchLabels["kubearmor.io/container.name"] = repotag[0]
+	// If user defined labels are present, update the matchLabels with them or use default matchLabels
+	if len(options.Uselabels) > 0 {
+		for _, uselabel := range options.Uselabels {
+			userLabel := strings.Split(uselabel, "=")
```

Comment:
Can we split either on the basis of "=" or ":' ?

I have seen folks getting confused as to what separator to use and the code can support either.

Reply:
Yes, I've updated the code to include both "=" and ":"